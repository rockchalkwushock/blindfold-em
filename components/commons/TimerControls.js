import Button from './Button'
import Fragment from './Fragment'

const TimerControls = ({ start, stop }) => (
  <Fragment>
    <Button className="start" fn={start} text="Start" />
    <Button className="stop" fn={stop} text="Stop" />
  </Fragment>
)

export default TimerControls
