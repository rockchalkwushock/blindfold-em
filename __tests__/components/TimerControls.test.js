import React from 'react'
import { mount } from 'enzyme'

import TimerControls from '../../components/commons/TimerControls'

describe('Component: <TimerControls />', () => {
  describe('Snapshots', () => {
    test('should mount without exploding as TimerControls', () => {
      const tree = mount(<TimerControls start={() => {}} stop={() => {}} />)
      expect(tree).toMatchSnapshot()
    })
  })
})
