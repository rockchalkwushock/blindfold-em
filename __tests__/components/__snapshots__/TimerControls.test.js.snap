// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: <TimerControls /> Snapshots should mount without exploding as TimerControls 1`] = `
<TimerControls
  start={[Function]}
  stop={[Function]}
>
  <Fragment>
    <Button
      className="start"
      fn={[Function]}
      text="Start"
    >
      <button
        className="jsx-383770121 start"
        onClick={[Function]}
      >
        Start
        <JSXStyle
          css="button.jsx-383770121{background-color:#734D84;border:none;box-shadow:2px 2px 2px #A974C2;color:#B0BF6C;font-family:\\"Kalam\\", cursive;font-size:1.5em;-webkit-letter-spacing:5px;-moz-letter-spacing:5px;-ms-letter-spacing:5px;letter-spacing:5px;margin:0.5em 0.25em 0 0.25em;outline:none;padding:0.25em;width:50%;}"
          styleId="383770121"
        />
      </button>
    </Button>
    <Button
      className="stop"
      fn={[Function]}
      text="Stop"
    >
      <button
        className="jsx-383770121 stop"
        onClick={[Function]}
      >
        Stop
        <JSXStyle
          css="button.jsx-383770121{background-color:#734D84;border:none;box-shadow:2px 2px 2px #A974C2;color:#B0BF6C;font-family:\\"Kalam\\", cursive;font-size:1.5em;-webkit-letter-spacing:5px;-moz-letter-spacing:5px;-ms-letter-spacing:5px;letter-spacing:5px;margin:0.5em 0.25em 0 0.25em;outline:none;padding:0.25em;width:50%;}"
          styleId="383770121"
        />
      </button>
    </Button>
  </Fragment>
</TimerControls>
`;
